scripts/upgrade-hooks/subgraph_config.json

```
@@ -0,0 +1,49 @@
+{
+  "testing": {
```

```suggestion
  "test": {
```
Just to be in line with how we store it in the addresses.

---

in core-components, we've always called our environments testing, staging and production
https://github.com/bosonprotocol/core-components/blob/ad1584da1f59383a7ce3581abbf8e6a663371474/packages/common/src/types/configs.ts#L15

---

Or we can change everything to "testing" here (and in Boson), I don't mind.

I just pointed it out since I noticed that the reading subgraph config is not compatibile with reading the addresses. Or we can introduce some remappings, but that it personally find ugly (I think I did sth similar somewhere because Boson uses "mainnet" and fermion uses "ethereum").

---

https://github.com/bosonprotocol/core-components/blob/ad1584da1f59383a7ce3581abbf8e6a663371474/packages/common/src/types/configs.ts#L37

---

^^ compared to the protocol deployment addresses files, we don't use any label for the chains, only chainId (less confusion possible)

---

FYI, I've added a reverse mapping data config in this script, but I agree this is a bit ugly: